addon/components/tf-form/component.js

```diff
@@ -0,0 +1,30 @@
+/**
+  @module ember-ticketfly-fields
+ */
+import Ember from 'ember';
+import layout from './template';
+const { Component, guidFor, computed } = Ember;
+
+/**
+  @public
+  @class TfForm
+  @extends Ember.Component
+ */
+export default Component.extend({
+  layout,
+  
+  tagName: 'form',
+  classNames: ['tf-form'],
+
+  inputGuid: computed({
+    get() {
+      return `id-${guidFor(this)}`;
+    }
+  }),
+
+  describedByGuid: computed({
+    get() {
+      return `described-by-${guidFor(this)}`;
+    }
+  })
+});
```

Comment:
I feel that the `form` component in your example above doesn't belong in this addon, as a form can hold many different types of inputs (submit, radio, checkbox) and this addon is only concerned with "fields". 

I think part of the confusion is because the design specs idea of a form (basically a text field with label or placeholder) is different from what we use form for semantic HTML.

On the same page, the design spec includes dropdown, date, and time inputs under "fields". We need to decide whether those should be separate addons or all bundled with this. If the former, we might consider renaming this addon to `ember-ticketfly-text-field`.

Then, for example: `ember-ticketfly-date-field` would include both the date input and the date picker functionality. On the other hand, a nitpicky thing is I don't like these verbose component names.

I think this warrants more fleshing out in RFCs for all the field types and I'll start writing that out. Let me know if you have any opinions on this. @BrianSipple @melissaroman @spencer516